README.md

```diff
@@ -42,13 +41,13 @@ into your HTML pages:
 Add this to your `<head>` element:
 
 ```html
-<link rel="stylesheet" href="/path/to/your/assets/css/uswds-0.9.0.css">
+<link rel="stylesheet" href="/path/to/your/assets/css/uswds.css">
```

Comment:
So should this line be more like:

`/path/to/your/assets/lib/uswds-0.9.0/cssuswds.css`